microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -72,12 +72,30 @@ To use Visual Studio 2017, you might also need to install a recent Windows SDK o
 <!-- ====================================================================== -->
 ## Step 4 - Open the solution in Visual Studio
 
-1. On your local drive, open the `.sln` file in Visual Studio:
+1. Open Visual Studio 2022.  (Or, if you have Visual Studio 2019 installed, you can use that version.)  For example, in Windows, in the Start menu, search for **Visual Studio 2022** app, and then click **Open**:
```

Comment:
I think this step is pretty verbose. We can just state
"Open Visual Studio 2022", and drop the screenshot

Reply:
reverted changes in PR other than debug article, will re-apply/ followup in other PR